src/js/options.js

```diff
@@ -0,0 +1,18 @@
+// Saves options to chrome.storage
+function save_options() {
```

Comment:
Just to be consistent with the rest of the code (probably will need adjustments where it is referenced)

```suggestion
const saveOptions = () => {
```